utils.css

```diff
@@ -339,6 +390,70 @@ label {
   background-color: var(--danger-dark);
 }
 
+/* badges */
+
+.badge-container {
+  position: relative;
+}
+
+.badge-container span {
+  background-color: var(--primary);
+  color: var(--white);
+  text-align: center;
+  position: absolute;
+  font-weight: bold;
+}
+
+.icon-badge {
+  width: 2rem;
+  height: 2rem;
+}
+
+.icon-badge span {
+  font-size: 0.675rem;
+  width: 1.325rem;
+  height: 1.325rem;
+  padding: 0.25rem;
+  top: -45%;
+  right: -30%;
+}
+
+/* cards */
+
+.card {
+  width: 15rem;
+  height: max-content;
+  border: 1px solid var(--light-grey);
+}
+.card:hover {
+  cursor: auto;
+}
+
+.card-shadow,
+.card:hover {
+  box-shadow: 13px 13px 21px -5px var(--box-shadow);
+}
```

Comment:
you can make separate class for shadow and use it here

Reply:
.card-shadow is the class for the box shadow. I am doing that only. This is the utils file and because i would want the same on hover Im using the same

Reply:
sure, go for it!